### ISSUE TITLE
Error: Can't connect to twilsock

### ISSUE THREAD

**Issue by varunlibra3** [twilio/TwilioChatJsReactNative#44]
Hi

I have been getting this error when I try to build the client.
Possible Unhandled Promise Rejection (id: 0): Error: Can't connect to twilsock

Once I get the token, I make a call TwilioChatClient.create(token) which returns the above error.
Interestingly, I copy and paste same token on Twilio sample Android app (https://github.com/twilio/twilio-chat-demo-android) like this:
ChatClient.create(context, token, props, this);
This completes and calls onSuccess(ChatClient chatClient)

So why it doesn't return the chatClient on React Native. Please help.

I am very sure that I am using live API credentials, not the test credentials.

**Comment by afreeland** [comment]
@varunlibra3 did you ever resolve this?  I am seeing this issue as well and I believe I am not using test credentials either.

**Comment by afreeland** [comment]
Not sure why its happening but everything was feeling like an invalid token.  I ended up removing the `ttl` and things started working for me?!

Removed this line:
https://github.com/twilio/TwilioChatJsReactNative/blob/c2cb6da9946a48997c5b31988a2b3069dfb63f49/js/token-provider.js#L26

**Comment by berkus** [comment]
@afreeland were you providing any value for  the customTTL before removing that line?

**Comment by afreeland** [comment]
@berkus I don't believe I was. If I remember correctly it just was _broke_ and I started throwing all sorts of things at it and then dug in to discover the `ttl` issue.

Ultimately we decided that there was not very much info (outside of this single repo) around RN and Twilio...and that although their docs have lots of information it still felt challenging/frustrating.  Went with [RN Firebase](https://rnfirebase.io/) and never looked back, their docs and guides are absolutely incredible! 💯
